compact/materials.xml

```diff
@@ -479,6 +484,13 @@
     <fraction n="0.70" ref="Cu"/>
     <fraction n="0.30" ref="Zn"/>
   </material>
+  <material name="BrassAlloy360">
```

Comment:
Suggested name: `Brass360` similar to `Steel235` etc. All are alloys.

Reply:
```suggestion
  <material name="Brass360">
```

Reply:
Presumably it's used elsewhere too, though?